From dcad5af090528018599277dc5d7e160fb6b2d68e Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 15 Jan 2020 20:26:40 +0000
Subject: [PATCH] Fix shim header generation when unbundling ICU

listformatter.h was moved from icuuc to icui18n

Bug: 989153
Change-Id: I9fcb56f6d5af7787f34ea99b737e2ed8fe741c84
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2000142
Reviewed-by: Lei Zhang <thestig@chromium.org>
Commit-Queue: Lei Zhang <thestig@chromium.org>
Cr-Commit-Position: refs/heads/master@{#732114}
---
 AUTHORS                     | 1 +
 build/linux/unbundle/icu.gn | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/AUTHORS b/AUTHORS
index 7523e483aae..1618fddc633 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -891,6 +891,7 @@ Soren Dreijer <dreijerbit@gmail.com>
 Sreerenj Balachandran <sreerenj.balachandran@intel.com>
 Srirama Chandra Sekhar Mogali <srirama.m@samsung.com>
 Staphany Park <stapark008@gmail.com>
+Stephan Hartmann <stha09@googlemail.com>
 Stephen Searles <stephen.searles@gmail.com>
 Steve Sanders <steve@zanderz.com>
 Steven Pennington <spenn@engr.uvic.ca>
diff --git a/build/linux/unbundle/icu.gn b/build/linux/unbundle/icu.gn
index 923bd7f5ac3..e77bc43db87 100644
--- a/build/linux/unbundle/icu.gn
+++ b/build/linux/unbundle/icu.gn
@@ -92,6 +92,7 @@ shim_headers("icui18n_shim") {
     "unicode/fpositer.h",
     "unicode/gender.h",
     "unicode/gregocal.h",
+    "unicode/listformatter.h",
     "unicode/measfmt.h",
     "unicode/measunit.h",
     "unicode/measure.h",
@@ -174,7 +175,6 @@ shim_headers("icuuc_shim") {
     "unicode/icudataver.h",
     "unicode/icuplug.h",
     "unicode/idna.h",
-    "unicode/listformatter.h",
     "unicode/localpointer.h",
     "unicode/locdspnm.h",
     "unicode/locid.h",
